tests/Doctrine/Migrations/Tests/Configuration/ConfigurationTest.php

```diff
@@ -122,7 +133,10 @@ public function testVersionsTryToGetLoadedIfNotAlreadyLoadedWhenAccessingMethodT
         );
         $migrator->migrate('3Test');
 
-        $result = call_user_func_array([$configuration, $method], $args);
+        /** @var callable $callable */
+        $callable = [$configuration, $method];
+
+        $result = call_user_func_array($callable, $args);
```

Comment:
I think this would be better (readability and performance) and would allow us to remove that `/** @var callable $callable */` which is never reliable:

```php
$result = $configuration->$method(...$args);
```

Reply:
```php
[$configuration, $method](...$args)
```
This should work too.